scripts/install.sh

```diff
@@ -30,3 +30,13 @@ tee /etc/apt/sources.list.d/docker.list > /dev/null
 VERSION_STRING=$1
 apt-get -qq update
 apt-get -qq install docker-ce="$VERSION_STRING" docker-ce-cli="$VERSION_STRING" containerd.io docker-buildx-plugin docker-compose-plugin
+
+#create/enable swapping
```

Comment:
I would wrap whole swap setup in a condition, like
if ! grep -q "swap" /etc/fstab ; then
...
fi
As I don't think all these commands are idempotent (e.g. fallocate on a file that's already in use)